test/higher_order_ops/test_with_effects.py

```diff
@@ -199,6 +213,130 @@ def f(x):
 
         res.sum().backward()
 
+    def test_register_effectful_custom_op(self):
+        lib = torch.library.Library("mylib", "FRAGMENT")  # noqa: TOR901
+        try:
```

Comment:
use torch.library._scoped_library

Reply:
I just copied it from [other test cases](https://www.internalfb.com/code/search?q=regex%3Aon%20repo%3Afbcode%20torch.library.Library.*%23%20noqa%3A%20TOR901) 😅 but now I know!